R-packages/covidcast/_pkgdown.yml

```diff
@@ -8,3 +8,43 @@ home:
 
     - text: View the COVIDcast map
       href: https://covidcast.cmu.edu/
+
+reference:
+  - title: Fetch data
+    desc: Fetch data from the COVIDcast API
+  - contents:
+      - covidcast_signal
+      - covidcast_signals
+      - covidcast_meta
+  - title: Analyze and plot data
+    desc: Plot and summarize signals
+  - contents:
+      - plot.covidcast_signal
+      - print.covidcast_signal
+      - print.covidcast_meta
+      - summary.covidcast_signal
+      - summary.covidcast_meta
+  - title: Wrangle and correlate multiple signals
+    desc: Calculate correlations and move signals between formats for analysis
```

Comment:
I might suggest: 
title: Wrangle and correlate multiple signals
desc: Calculate correlations, aggregate signals, and move between aggregated data formats